analysis_driver/report_generation/report_crawlers.py

```diff
@@ -22,7 +22,27 @@
     ELEMENT_GENOTYPE_VALIDATION, ELEMENT_COVERAGE_STATISTICS, ELEMENT_MEAN_COVERAGE, ELEMENT_COVERAGE_PERCENTILES, \
     ELEMENT_BASES_AT_COVERAGE, ELEMENT_MEDIAN_COVERAGE_SAMTOOLS, ELEMENT_COVERAGE_SD, ELEMENT_FREEMIX, ELEMENT_SAMPLE_CONTAMINATION, \
     ELEMENT_GENDER_VALIDATION, ELEMENT_GENDER_HETX, ELEMENT_LANE_PC_OPT_DUP, ELEMENT_GENDER_COVY, ELEMENT_SNPS_TI_TV, \
-    ELEMENT_SNPS_HET_HOM
+    ELEMENT_SNPS_HET_HOM, ELEMENT_SAMPLE_PLATE, ELEMENT_SAMPLE_SPECIES, ELEMENT_SAMPLE_EXPECTED_YIELD, ELEMENT_SAMPLE_EXPECTED_COVERAGE
+
+
+
+def get_sample_information_from_lims(sample_name):
+    lims_sample = clarity.get_sample()
+    gender = SampleCrawler.gender_aliases(clarity.get_sample_gender())
```

Comment:
This will raise `TypeError: 'dict' object is not callable`, and no sample name passed to `get_sample_gender`, as below:
`gender = SampleCrawler.gender_aliases[clarity.get_sample_gender(sample_name)]`

It would probably also make sense to move the gender alias out of `SampleCrawler` back to the top level.